Classes/ViewHelpers/Format/ICalendarDescriptionViewHelper.php

```diff
@@ -52,10 +52,10 @@ public function render(): string
         // Strip new lines
         $tmpDescription = str_replace(chr(10), '', $tmpDescription);
         // Glue everything together, so every line is max 75 chars
-        if (strlen($tmpDescription) > 75) {
-            $newDescription = substr($tmpDescription, 0, 63) . chr(10);
-            $tmpDescription = substr($tmpDescription, 63);
-            $arrPieces = str_split($tmpDescription, 74);
+        if (mb_strlen($tmpDescription) > 63) {
+            $newDescription = mb_substr($tmpDescription, 0, 63) . chr(10);
+            $tmpDescription = mb_substr($tmpDescription, 63);
+            $arrPieces = mb_str_split($tmpDescription, 75);
```

Comment:
Switching to max chars 75 and afterwards an additional whitspace is added resulting in 76 chars -  like Google Calendar.